fairing/preprocessors/full_notebook.py

```diff
@@ -20,12 +21,17 @@ def __init__(self,
         if notebook_file is None:
             raise ValueError('A notebook_file must be provided.')
 
+        relative_notebook_file = notebook_file
+        # Convert absolute notebook path to relative path
+        if notebook_file[0] == '/':
```

Comment:
This will break in non-posix systems. use https://docs.python.org/3/library/os.path.html#os.path.isabs ?

Reply:
Done